diff --git a/applets/systemtray/qml/ExpandedRepresentation.qml b/applets/systemtray/qml/ExpandedRepresentation.qml
--- a/applets/systemtray/qml/ExpandedRepresentation.qml
+++ b/applets/systemtray/qml/ExpandedRepresentation.qml
@@ -242,7 +242,10 @@ Item {
                 KeyNavigation.left: actionsButton.visible ? actionsButton : actionsButton.KeyNavigation.left
                 KeyNavigation.right: pinButton
 
-                PlasmaComponents.ToolTip {